tests/Unit/NumericalAlgorithms/RootFinding/Test_OneDRootFinder.cpp

```diff
@@ -104,15 +107,15 @@ SPECTRE_TEST_CASE("Unit.Numerical.RootFinding.TOMS748RootSolver.DataVector",
     return constant[i] - x * x;
   };
 
-  find_root_of_function(f_lambda, lower, upper, abs_tol, rel_tol);
+  RootFinder::secant(f_lambda, lower, upper, abs_tol, rel_tol);
   ERROR("Failed to trigger ASSERT in an assertion test");
 #endif
 }
 
-// [[OutputRegex, The relative tolerance is too small.]]
-[[noreturn]] SPECTRE_TEST_CASE(
-    "Unit.Numerical.RootFinding.TOMS748RootSolver.RelativeTol.Double",
-    "[NumericalAlgorithms][RootFinding][Unit]") {
+    // [[OutputRegex, The relative tolerance is too small.]]
```

Comment:
undo this - no spaces here please - also rename the test to secant.
check through the rest of the file as well, there should be no mention of TOMS left after this rename